build.gradle.kts

```diff
@@ -91,7 +84,9 @@ application {
 }
 
 detekt {
-    failFast = true // fail build on any finding
+    // For now only consider main source files, but ignore test sources
+    input = files("src/main/kotlin")
```

Comment:
Detekt now by default also checks test sources (see https://github.com/detekt/detekt/pull/3649), that causes multiple findings for Arisa due to too long lines and too large classes, so for now it might be best to not check test sources.

Maybe in the future we could [configure](https://detekt.github.io/detekt/configurations.html) Detekt to disable checks such as `style` in test sources instead, but still run the other checks (to detect broken tests).